### PR TITLE
feat/shimmer-quiz

### PR DESCRIPTION
**Pull request by huhn511**
# Description of change

add shimmer quiz with several questions.

### CODE REVIEW

File: shimmer/learn/shimmer-questions.json [hunk 1]
```diff
@@ -0,0 +1,83 @@
+[
+    {
+      "questionText": "The staging and innovation network of IOTA is called ...?",
+      "answerOptions": [
+        { "answerText": "Assembly", "isCorrect": false },
+        { "answerText": "Soon Network", "isCorrect": false },
```
Comment (jlvandenhout) [inline shimmer/learn/shimmer-questions.json:6]:
```suggestion
        { "answerText": "Soon network", "isCorrect": false },
```

File: shimmer/learn/shimmer-questions.json [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        { "answerText": "IOTA 2.0", "isCorrect": false },
+        { "answerText": "Shimmer", "isCorrect": true }
+      ]
+    },
+    {
+      "questionText": "What's the beginning of a Shimmer Testnet address?",
```
Comment (jlvandenhout) [inline shimmer/learn/shimmer-questions.json:12]: AFAIK it's iether the Shimmer Beta network or just Testent right? As Shimmer is now live, I'd say we stick to just Testnet.

```suggestion
      "questionText": "What's the beginning of a Testnet address?",
```

File: shimmer/learn/shimmer-questions.json [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+      "answerOptions": [
+        { "answerText": "smrtest", "isCorrect": false },
+        { "answerText": "rms", "isCorrect": true },
+        { "answerText": "shmtest", "isCorrect": false },
+        { "answerText": "mhs", "isCorrect": false }
+      ]
+    },
+    {
+      "questionText": "How many bytes can be used for the immutable data output, which is used for example for NFT Metadata?",
```
Comment (jlvandenhout) [inline shimmer/learn/shimmer-questions.json:21]:
```suggestion
      "questionText": "How many bytes can be used for the immutable data output, which is used for example for NFT metadata?",
```

File: shimmer/learn/shimmer-questions.json [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+      "questionText": "What is a DEX?",
+      "answerOptions": [
+        { "answerText": "Dungeon Entry 10", "isCorrect": false },
+        { "answerText": "Disturbed Experience", "isCorrect": false },
+        { "answerText": "Decentralised Exchange", "isCorrect": true },
+        { "answerText": "Distributed Edge Cross", "isCorrect": false }
+      ]
+    },
+    {
+      "questionText": "What does DAO stands for?",
```
Comment (jlvandenhout) [inline shimmer/learn/shimmer-questions.json:57]:
```suggestion
      "questionText": "What does DAO stand for?",
```

File: shimmer/learn/shimmer-questions.json [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+        { "answerText": "Teapot", "isCorrect": false },
+        { "answerText": "Exchange", "isCorrect": false }
+      ]
+    },
+    {
+      "questionText": "What is a DEX?",
+      "answerOptions": [
+        { "answerText": "Dungeon Entry 10", "isCorrect": false },
+        { "answerText": "Disturbed Experience", "isCorrect": false },
+        { "answerText": "Decentralised Exchange", "isCorrect": true },
```
Comment (jlvandenhout) [inline shimmer/learn/shimmer-questions.json:52]:
```suggestion
        { "answerText": "Decentralized Exchange", "isCorrect": true },
```

We converged to American spelling.

File: shimmer/learn/shimmer-questions.json [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+        { "answerText": "Distributed Edge Cross", "isCorrect": false }
+      ]
+    },
+    {
+      "questionText": "What does DAO stands for?",
+      "answerOptions": [
+        { "answerText": "Don't Allow Onigrams", "isCorrect": false },
+        { "answerText": "Decentralised Autonomous Organisation", "isCorrect": true },
+        { "answerText": "Decentralised Aggregated Organisation", "isCorrect": false },
```
Comment (jlvandenhout) [inline shimmer/learn/shimmer-questions.json:61]:
```suggestion
        { "answerText": "Decentralized Autonomous Organisation", "isCorrect": true },
        { "answerText": "Decentralized Aggregated Organisation", "isCorrect": false },
```

File: shimmer/learn/shimmer-questions.json [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+        { "answerText": "Distributed Anywhere Organisation", "isCorrect": false }
+      ]
+    },
+    {
+      "questionText": "How do we call Tokens on Layer 1 (L1)?",
```
Comment (jlvandenhout) [inline shimmer/learn/shimmer-questions.json:66]:
```suggestion
      "questionText": "What do we call tokens on Layer 1 (L1)?",
```

File: shimmer/learn/shimmer-questions.json [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+      "answerOptions": [
+        { "answerText": "Natural Tokens", "isCorrect": false },
+        { "answerText": "New Tokens", "isCorrect": false },
+        { "answerText": "Native Tokens", "isCorrect": true },
+        { "answerText": "Necend Tokens", "isCorrect": false }
```
Comment (jlvandenhout) [inline shimmer/learn/shimmer-questions.json:71]:
```suggestion
        { "answerText": "Natural tokens", "isCorrect": false },
        { "answerText": "New tokens", "isCorrect": false },
        { "answerText": "Native tokens", "isCorrect": true },
        { "answerText": "Necend tokens", "isCorrect": false }
```

File: shimmer/learn/shimmer-questions.json [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+      ]
+    },
+    {
+      "questionText": "Shimmer is ...?",
+      "answerOptions": [
+        { "answerText": "Feeless", "isCorrect": true },
+        { "answerText": "Free", "isCorrect": false },
+        { "answerText": "Decentral", "isCorrect": false },
+        { "answerText": "a Blockchain", "isCorrect": false }
```
Comment (jlvandenhout) [inline shimmer/learn/shimmer-questions.json:80]:
```suggestion
        { "answerText": "A blockchain", "isCorrect": false }
```

**Review by jlvandenhout — CHANGES_REQUESTED** [review]

**Review by Dr-Electron — APPROVED** [review]

**Pull request merged**